vtex/faststore
# adjust sku validation

## What's the purpose of this pull request?

Need to adjust the SKU validation because when selecting an SKU with the name "7", for example, the SKU with the name "7.5" is being selected instead of "7".

## How it works?

The adjustment involves replacing the code that previously used slugs.includes() with strict equality slugs ===

## How to test it?

On a product page with SKUs, if you select one named '7', you'll notice that it selects SKU '7.5' instead.
